src/screens/Dashboard/screens/Login/index.tsx

```diff
@@ -1,7 +1,51 @@
+import { ChangeEvent, useState } from 'react';
+
+import styles from './styles.module.scss';
+import logo from './assets/LogoWolox.png';
+import { dataForm } from './constants';
+
 function Login() {
+  const [user, setUser] = useState({
+    email: '',
+    password: '',
+    passwordConfirmation: '',
+    firstName: '',
+    lastName: '',
+    locale: 'en'
+  });
+
+  const changeValue = (event: ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+    setUser({ ...user, [name]: value });
+  };
+
+  const printData = () => {
+    /* console.log(user); */
+  };
+  // crear en la carpeta constants dentro de Login
   return (
-    <div>
-      <span>login</span>
+    <div className={styles.app}>
+      <form className={styles.rectangle}>
+        <div className={styles.head} />
+        <img src={logo} className={styles.logoWolox} alt="logo" />
+
+        <label>
+          {dataForm.map((data) => (
+            <div key={data.name}>
+              <p className={styles.textLabel}>{data.label}</p>
+              <input type={data.type} name={data.name} className={styles.rectangle1} onChange={changeValue} />
+            </div>
+          ))}
+        </label>
+
+        <button type="button" onClick={printData} className={styles.group1}>
```

Comment:
- same css class comment
```suggestion
        <button type="button" onClick={printData} className={styles.btn-primary}>
```